29.md

```diff
@@ -40,7 +40,7 @@ Relays should prevent late publication (messages published now with a timestamp
 
 - *text root note* (`kind:11`)
 
-This is the basic unit of a "microblog" root text note sent to a group.
+This is the basic unit of a "forum" root text note sent to a group. These should likely be seperated from `kind:9` & `kind:10` events as mixing the two could be quite confusing. 
```

Comment:
What is `kind:10`?

Reply:
Chat message threaded reply?

Reply:
kinds 9,10 dealing with chat messages and kinds 11,12 dealing with forum messages? no?

Reply:
Yes, sorry, I missed that in the text.